docs/source/examples/predict_bike_trips.ipynb

```diff
@@ -0,0 +1,414 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Predict Bike Trips\n",
+    "\n",
+    "In this example, we build a machine learning application to predict the number of bike trips from a station in the next biking period. This application is structured into three important steps:\n",
+    "\n",
+    "* Prediction Engineering\n",
+    "* Feature Engineering\n",
+    "* Machine Learning\n",
+    "\n",
+    "In the first step, we generate new labels from the data by using [Compose](https://compose.alteryx.com/). In the second step, we generate features for the labels by using [Featuretools](https://docs.featuretools.com/). In the third step, we search for the best machine learning pipeline by using [EvalML](https://evalml.alteryx.com/). \n",
+    "After working through these steps, you will learn how to build machine learning applications for real-world problems like forecasting demand. Let's get started."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from demo.chicago_bike import load_sample\n",
+    "from matplotlib.pyplot import subplots\n",
+    "import composeml as cp\n",
+    "import featuretools as ft\n",
+    "import evalml"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will use data provided by Divvy which is a bicycle sharing system in Chicago. In this dataset, we have a record of each bike trip."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df = load_sample()\n",
+    "\n",
+    "df.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prediction Engineering\n",
+    "\n",
+    "> How many trips will occur from a station in the next biking period?\n",
+    "\n",
+    "We can change the length of the biking period to create different prediction problems. For example, how many bike trips will occur in the next 4 hours or in the next week? These variations can be done by simply tweaking a parameter. This helps us explore different scenarios which is crucial for making better decisions.\n",
+    "\n",
+    "### Defining the Labeling Process\n",
+    "\n",
+    "Let's start by defining a labeling function to calculate the number of trips. Given that each observation is an individual trip, the number of trips is just the number of observations."
```

Comment:
Might add a sentence to explain that this labeling function will be called by the `LabelMaker` when executing the search to create the labels. That could be helpful for new users that don't fully understand the steps in Compose yet